x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -298,6 +314,19 @@ function FieldPanelHeader({
         dropOntoWorkspace={dropOntoWorkspace}
         field={draggableField}
       />
+      {editField && (
+        <EuiButtonIcon
+          onClick={() => editField(field.name)}
+          iconType="pencil"
+          data-test-subj="lnsFieldListPanelEdit"
+          title={i18n.translate('xpack.lens.indexPattern.editFieldLabel', {
+            defaultMessage: 'Edit field in index pattern',
```

Comment:
Rather than using the `title` prop here, can we use a tooltip for this button (as we currently do for the adjacent button to add to workspace)?